Source/WTF/wtf/cocoa/SoftLinking.h

```diff
@@ -28,6 +28,9 @@
 #import <objc/runtime.h>
 #import <wtf/Assertions.h>
 
+#define _STORE_IN_DLSYM_SECTION __attribute__((section("__TEXT,__dlsym_cstr")))
```

Comment:
This got merged already, but I'm surprised by the leading underscore. Why not `WTF_STORE_IN_DLSYM_SECTION`?

Reply:
My thinking was to indicate that this macro is only for use inside of `SoftLinking.h`. We have some limited precedent for this pattern in other WTF headers like `Compiler.h`.